frontend/src/pages/ChatPage/components/ReportModal.tsx

```diff
@@ -72,6 +82,11 @@ export default function ReportModal(props: {
               return
             }
 
+            // In after reviewed report(called System report) has only 2 choices: no-show and others
+            if (props.isSystemReport && data.type != "no-show" && data.type != "others") {
+              return
+            }
```

Comment:
ลองเปลี่ยนมาใช้พวก built-in function เช่น .filter น่าจะทำให้สวยขึ้นนะ 
[https://medium.com/poka-techblog/simplify-your-javascript-use-map-reduce-and-filter-bd02c593cc2d](url)